src/Util.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace React\Http;
+
+/**
+ * @internal
+ */
+final class Util
+{
+    /**
+     * Convert a ini like size to a numeric size in bytes.
+     *
+     * @param string $iniSetting
+     * @return int
+     */
+    public static function iniSizeToBytes($size)
+    {
+        $suffix = strtoupper(substr($size, -1));
+        $strippedSize = substr($size, 0, -1);
+
+        if (!is_numeric($strippedSize)) {
+            throw new \InvalidArgumentException('"' . $strippedSize . '" is not a valid ini size');
```

Comment:
The exception message should probably use the passed `$size`.

Reply:
Also, that fails for single-digits.

Reply:
Cheers, fixed that :+1:

Reply:
And also addressed the single digit issue :+1: